src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -106,7 +108,8 @@ public ApiSpecification(ITestOutputHelper output) : base(output)
         protected override void BeforeTest()
         {
             this.sharedSteps = new SharedSteps();
-            this.httpClient = new HttpClient();
+            this.httpHandler = new HttpClientHandler() { ServerCertificateCustomValidationCallback = (request, cert, chain, errors) => true };
```

Comment:
This callback always returns true.  Is this work in progress ?

Reply:
no this is to bypass the "invalid certificate" errors you receive by using a certificate that is not in the "Root" store.
for tests, we don't put the certificate in the "Root" store because you get a windows pop up warning when you do that for the first time

Reply:
Ar okay, thanks.